src/Discord/Internal/Types/ScheduledEvents.hs

```diff
@@ -0,0 +1,188 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+
+-- | Structures pertaining to Discord Scheduled Events
```

Comment:
:+1:  happy to see a doc

Reply:
I try too put them 
Hesitating to copy the field descriptions from the discord documentation as doc for the fields of the types